src/www/ui/template/include/macros.html.twig

```diff
@@ -34,7 +34,7 @@
     {% set size = 0 %}
   {% endif %}
   <select name="{{ name }}"{% if id is defined %} id="{{ id }}"{% endif %}{% if size > 0  %} multiple="multiple" size="{{ size }}"{% endif %}{% if action is defined %}
-    {{ action}}{% endif %} class="ui-render-select2">
```

Comment:
> ui-render-select2

Here is adding the select2 search for all the select tag's. there are many which has only 2 to 5 values which doesn't need any search. that's  why this class is removed from here.

We can find the place where it is really needed instead to adding it to every select tag.

Reply:
> Here is adding the select2 search for all the select tag's. there are many which has only 2 to 5 values which doesn't need any search. that's why this class is removed from here.

Adding select2 to a drop-down with only 2-5 values does nothing wrong but enhancing the visual appearance of the UI and make it look much more consistent with other drop down.
 
> We can find the place where it is really needed instead to adding it to every select tag.

Adding the class to every place where it is required will be a laborious job. That's why it was add to the macro itself.

Reply:
>>Here is adding the select2 search for all the select tag's. there are many which has only 2 to 5 values which doesn't need any search. that's why this class is removed from here.

>Adding select2 to a drop-down with only 2-5 values does nothing wrong but enhancing the visual appearance of the UI and make it look much more consistent with other drop down.

i don't think visual appearance get enhances but for some browsers it will not show the selected value[attached image]. so better it should be removed from all the places and apply where required.
![with withoutSELECT2](https://user-images.githubusercontent.com/10155976/58539743-9bb89480-8215-11e9-88d4-51509f48b52d.PNG)


>>We can find the place where it is really needed instead to adding it to every select tag.

>Adding the class to every place where it is required will be a laborious job. That's why it was add to the macro itself.

We already have Select2 in almost all the places. just needs to add in in few more places. which will not consume much time i think.

Reply:
> i don't think visual appearance get enhances but for some browsers it will not show the selected value[attached image]. so better it should be removed from all the places and apply where required.
> ![with withoutSELECT2](https://user-images.githubusercontent.com/10155976/58539743-9bb89480-8215-11e9-88d4-51509f48b52d.PNG)

There is an open issue for the same (#1300) which is fixed by #1329.

> We already have Select2 in almost all the places. just needs to add in in few more places. which will not consume much time i think.

If you still prefer, please go ahead.